install-dev/models/install.php

```diff
@@ -114,21 +116,14 @@ public function generateSettingsFile(
                     'database_engine' =>  $database_engine,
                     'cookie_key' => $cookie_key,
                     'cookie_iv' =>  $cookie_iv,
+                    'new_cookie_key' => $key->saveToAsciiSafeString(),
```

Comment:
It's not a salt and isn't `cookie_key_2` just as ugly? :stuck_out_tongue_winking_eye: